src/test/java/com/google/edith/UserInsightsTest.java

```diff
@@ -158,4 +161,61 @@ public void aggregateData() {
    
     Assert.assertEquals(map, userInsights.aggregateUserData());
   }
+
+//   @Test
```

Comment:
I am going to refactor this test.

Reply:
Can you add a TODO for this ?

Reply:
I already updated it in the newest commit.